packages/abbr/src/abbr.test.ts

```diff
@@ -1,12 +1,16 @@
+import { describe, test } from 'node:test'
+import assert from 'node:assert/strict'
+
 import abbr from './abbr'
 
 describe('abbr', () => {
   test('main', async () => {
-    expect(
+    assert.equal(
```

Comment:
nit: it'd be nice to be consistent (either always use `assert.equal`, or always use `assert.strictEqual`). Note that this won't change anything to the test results since you're using `node:assert/strict`, it's just a matter of readability.